README.md

```diff
@@ -4,7 +4,12 @@ The main credential are automatically set up for you, howver, the **databaseURL*
 Use the following command to set the **databaseURL** & the **storageBucket** URLs:
   
 ```sh
-$ firebase functions:config:set audioseva.database_url="THE DATABASE URL" audioseva.storage_bucket="THE STORAGE BUCKET URL"
+$ firebase functions:config:set audioseva.database_url="THE DATABASE URL"
```

Comment:
I'm not sure we should use `audioseva`prefix, as all this is about audioseva. Is it a good practice that you learned in Firebase doc?

Reply:
No not really, in fact they suggested prepending the key with the service name ( this only applies to the sendInBlue Key ) .. So I think it really needs restructuring.